src/wix/WixToolset.Core/Link/AddRequiredStandardDirectories.cs

```diff
@@ -43,12 +43,17 @@ public void Execute()
 
                 if (String.IsNullOrEmpty(parentDirectoryId))
                 {
-                    if (directory.Id.Id != "TARGETDIR")
-                    {
-                        directory.ParentDirectoryRef = "TARGETDIR";
-                    }
+                    parentDirectoryId = this.GetStandardDirectoryParent(directory.Id.Id, platform);
+
+                    directory.ParentDirectoryRef = parentDirectoryId;
+
+                    //if (directory.Id.Id != "TARGETDIR")
```

Comment:
etc

Reply:
This was all removed in a subsequent commit.